src/overlays/actors/ovl_En_Clear_Tag/z_en_clear_tag.h

```diff
@@ -8,9 +8,82 @@ struct EnClearTag;
 
 typedef struct EnClearTag {
     /* 0x0000 */ Actor actor;
-    /* 0x014C */ char unk_14C[0xB8];
+    /* 0x014C */ u8 shouldExplode;
+    /* 0x014D */ u8 drawMode;
+    /* 0x014E */ u8 state;
+    /* 0x0150 */ s16 work[3];
+    /* 0x0158 */ Vec3f targetPosition;
+    /* 0x0164 */ Vec3f targetDirection;
+    /* 0x0170 */ Vec3f acceleration;
+    /* 0x017C */ u8 timer;
+    /* 0x017D */ u8 shouldShootLaser;
+    /* 0x0180 */ f32 roll;
+    /* 0x0184 */ s16 crashingTimer;
+    /* 0x0186 */ s16 deathTimer;
+    /* 0x0188 */ Vec3f floorTangent;
+    /* 0x0194 */ ColliderCylinder collider;
+    /* 0x01E0 */ u8 demoMode;
+    /* 0x01E2 */ s16 cameraNumber;
+    /* 0x01E4 */ Vec3f demoCameraAt;
+    /* 0x01F0 */ Vec3f demoCameraEye;
+    /* 0x01FC */ s16 cutsceneTimer;
+    /* 0x01FE */ UNK_TYPE1 unk_1FE[0x06];
```

Comment:
It needs to be there or else the struct size doesn't match. Padding out put the full struct size at 0x200. There is probably 2 bytes of padding an unused var.

Reply:
`char` arrays are what we normally use for fillers, while `UNK_TYPE1` is mostly intended for a variable of size 1 byte and unknown signedness, so I would just use `char` here

Reply:
`UNK_TYPE1 is mostly intended for a variable of size 1 byte and unknown signedness`
This description fits here. We don't know what signedness this data originally had.

Unfortunately we don't know the size of this value either, so `char` doesn't technically fit. I think it's better to use some kind of UNK_TYPE to flag to someone reading that we just don't know what it's type originally was.

Reply:
We use `char` arrays for fillers in literally every other case, so I don't see why it would be different here...

It's going to be pretty clear to anyone reading the codebase that `char unk_XXX[...]` is just random space we don't know about and isn't used, since that's we do everywhere and other decomp projects do the same or something very similar.